### ISSUE TITLE
CRITICAL: OneSignal blocks applicationDidBecomeActive

### ISSUE THREAD

**Issue by markuswinkler** [OneSignal/OneSignal-iOS-SDK#55]
Version 2 prevents
```
func applicationDidBecomeActive(application: UIApplication) {
    //  important custom code
}
```
in `AppDelegate` from firing. 1.13.3 works.

**Comment by JKalash** [comment]
Hi,

Which version of the 2.* SDK are you using?

**Comment by markuswinkler** [comment]
latest one, 2.0.8

**Comment by JKalash** [comment]
Not being able to reproduce the bug. Could you provide me with:
- iOS version
- Device? / simulator?
- method calls to OneSignal singleton made in the initWithLaunchOptions?

**Comment by markuswinkler** [comment]
I made a barebones github repository to demonstrate:
https://github.com/markuswinkler/OneSignalBug

This is missing all the specific initCode for OneSignal. Just linking via CocoaPod already suppresses `applicationDidBecomeActive`

If you switch in the podfile from 2.0.8 to 1.13.3 it works.

- iOS version
8.1 - 9.3 (didn't matter)
- Device? / simulator?
iPhone / iPad

**Comment by JKalash** [comment]
Fix has been made. Will be released with OneSignal 2.0.9. Thanks for the feedback.

**Comment by markuswinkler** [comment]
Great! Thank you!

**Comment by JKalash** [comment]
Fixed [2.0.9](https://github.com/OneSignal/OneSignal-iOS-SDK/releases/tag/2.0.9)
